plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -211,8 +211,9 @@ export class EventsProcessor {
 
     public isNewPersonPropertiesUpdateEnabled(teamId: number): boolean {
         try {
-            const teamsStrs = this.pluginsServer.NEW_PERSON_PROPERTIES_UPDATE_ENABLED_TEAMS.split(',').filter(String)
-            const teams = teamsStrs.map((teamId) => Number(teamId))
+            const teams = this.pluginsServer.NEW_PERSON_PROPERTIES_UPDATE_ENABLED_TEAMS.split(',')
+                .filter(String)
```

Comment:
is this to remove stuff like `''`?

Reply:
jup the `.filter(String)` removes empty strings, which otherwise would convert to 0 from the map call. Which you told me can't be a team_id anyway, but if someone were to copy this code elsewhere it will work right.